mapie/classification.py

```diff
@@ -43,29 +43,30 @@ class MapieClassifier(BaseEstimator, ClassifierMixin):
         Method to choose for prediction interval estimates.
         Choose among:
 
-        - "naive", sum of the probabilities until the 1-alpha thresold.
+        - ``"naive"``, sum of the probabilities until the 1-alpha thresold.
 
-        - "score", based on the the scores
+        - ``"label"`` (formerly called ``"score"``), based on the the scores
           (i.e. 1 minus the softmax score of the true label)
           on the calibration set. See [1] for more details.
 
-        - "cumulated_score", based on the sum of the softmax outputs of the
+        - ``"aps"`` (formerly called "cumulated_score"), Adaptive Prediction
+          Sets method. It is based on the sum of the softmax outputs of the
           labels until the true label is reached, on the calibration set.
           See [2] for more details.
 
-        - "raps", Regularized Adaptive Prediction Sets method. It uses the
+        - ``"raps"``, Regularized Adaptive Prediction Sets method. It uses the
           same technique as cumulated_score method but with a penalty term
```

Comment:
aps

Reply:
Do you suggest to remove `raps` and remplace it with `aps`?